src/components/MediaPlayer/VideoJS/components/js/VideoJSProgress.js

```diff
@@ -274,17 +283,20 @@ function ProgressBar({
 
   React.useEffect(() => {
     setCanvasTargets(targets);
-    setCanvasTimes(targets[srcIndex]);
+    const cTimes = targets[srcIndex];
+    setCanvasTimes(cTimes);
     setActiveSrcIndex(srcIndex);
 
-    const right = canvasTargets.filter((_, index) => index > srcIndex);
-    const left = canvasTargets.filter((_, index) => index < srcIndex);
+    const right = canvasTargetsRef.current.filter((_, index) => index > srcIndex);
+    const left = canvasTargetsRef.current.filter((_, index) => index < srcIndex);
     setTRight(right);
     setTLeft(left);
 
-    const curTime = player.currentTime();
-    setProgress(curTime);
-    setCurrentTime(curTime + canvasTimes.altStart);
+    setProgress(cTimes.start);
+    setInitTime(cTimes.start);
+
+    setCurrentTime(cTimes.start);
+    player.currentTime(cTimes.start);
```

Comment:
Can you explain these changes?  I don't understand the change of setting currentTime to `cTimes.start` instead of `player.currentTime() + cTimes.altStart`.

Reply:
The `altStart` applies to the use-case with multiple files in a single Canvas (cookbook recipe 64). I tested these changes for that recipe just now and it seems this line didn't make a difference for that use-case, so we can remove this . I guess, this is one of the code lines we used in the initial setup stage and somehow gotten overridden in another place? However I noticed one tiny change needed for timeupdate, which I will commit right after this.